java/arcs/core/storage/driver/Volatile.kt

```diff
@@ -47,6 +47,14 @@ data class VolatileDriverProvider(private val arcId: ArcId) : DriverProvider {
     override suspend fun removeEntitiesCreatedBetween(startTimeMillis: Long, endTimeMillis: Long) =
         // Volatile storage is opaque, so remove all entities.
         removeAllEntities()
+
+    override suspend fun getEntitiesCount(inMemory: Boolean): Long {
+        if (inMemory) {
+            return arcMemory.size().toLong()
```

Comment:
this is not really the entity count, arcMemory will contain an entry for each storage key. A storage key can be an entity or a container (collection or singleton). Unfortunately Volatile does not know whether an entry is an entity or a container.

Reply:
I just reverted this part. We shouldn't count them as entities.